best-practice: update massive regions to match 4.0 (#2595)
pingcap/docs: best-practices/massive-regions-best-practices.md

![Check pd-worker](/media/best-practices/pd-worker-metrics.png)

> We have optimized pd-worker for better performance since [v3.0.5](/releases/release-3.0.5.md#tikv). If you encounter a similar problem, it is recommended to upgrade to a latest version.

Reviewer: 技术文档中应当避免使用第一人称`我`或`我们`。
```suggestion
pd-worker has been optimized for better performance since [v3.0.5](/releases/release-3.0.5.md#tikv). If you encounter a similar problem, it is recommended to upgrade to the latest version.
```
